src/components/Questions/QuestionModal.vue

```diff
@@ -208,6 +202,7 @@ export default defineComponent({
     }
 
     function showNextQuestion() {
+      // we reRender the whole component as textarea is holding the details which is not getting updated
```

Comment:
Also, in the placeholder, should we use something like 
`Enter your answer here. Only numbers allowed`?
Because I noticed that this just looks like a subjective answer.
And if someone is coming to the tool for the first time, they might not know about the question being a numerical question and they would be trying to type alphabets but nothing would happen

Reply:
<img width="537" alt="Screenshot 2022-05-31 at 4 09 59 PM" src="https://user-images.githubusercontent.com/20169829/171155264-e7251611-6425-4236-8386-179f90186101.png">

If I submit an answer, let's say `0`, it goes to the backend as an integer.
But when I reload, and the session answers are received by frontend from the backend, the value coming is a string `"0"`.
Ideally it should be an integer/float. This is happening because of the screenshot attached above. You should also updated `answerType` in the backend so that answers returned by the backend can be of the form integer/float.
I guess you can just do float. And depending on the type of the question on the frontend, (int or float), you can cast the incoming value to one with decimals or not